lbz/events/api.py

```diff
@@ -0,0 +1,58 @@
+import json
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from mypy_boto3_events.type_defs import PutEventsRequestEntryTypeDef
+else:
+    PutEventsRequestEntryTypeDef = dict
+
+from os import getenv
+from typing import List
+
+from lbz.boto3_client import clients
+from lbz.misc import Singleton
+
+
+class BaseEvent:
+    type: str
+
+    def __init__(self, raw_data: dict) -> None:
+        self.raw_data = raw_data
+        self.data: str = self.serialize(raw_data)
+
+    @staticmethod
+    def serialize(data: dict) -> str:
+        return json.dumps(data, default=str)
+
+
+class EventAPI(metaclass=Singleton):
```

Comment:
I think it might be named `EventBridge` or `EventBus`, don't you think? Just a suggestion as API here might be misunderstood.

Reply:
As we agreed, we keep the current name, at least for now.